desktop_server/src/com/mygdx/game/server/ecs/entityfactory/ComponentFactory.java

```diff
@@ -89,6 +92,13 @@ public void createCoordinateComponent(Coordinates coordinates, int entityId) {
     position.setCoordinates(coordinates);
   }
 
+  public void createMoveRangeComponent(int moveRange, int entityId){
+    var range = moveRangeMapper.create(entityId);
+    range.setMoveRange(moveRange);
+    //TODO: Może current inicjalizowany od 0?
```

Comment:
to zależy od tego kiedy będziemy resetować punkty ruchu, ale raczej nie trzeba narazie